app/code/Magento/StoreGraphQl/Model/Resolver/Store/StoreConfigDataProvider.php

```diff
@@ -88,9 +87,9 @@ private function getBaseConfigData(StoreInterface $store) : array
             'secure_base_url' => $storeConfig->getSecureBaseUrl(),
             'secure_base_link_url' => $storeConfig->getSecureBaseLinkUrl(),
             'secure_base_static_url' => $storeConfig->getSecureBaseStaticUrl(),
-            'secure_base_media_url' => $storeConfig->getSecureBaseMediaUrl()
+            'secure_base_media_url' => $storeConfig->getSecureBaseMediaUrl(),
+            'store_name' => $storeConfig->getStoreName()
```

Comment:
While you're changes look correct, I just wonder if we could do it in a less disruptive way. The current approach introduces a backwards incompatible change because we're adding a new public method to an @api interface.

Could we just use $store->getName() here and avoid the changed outside of GrapohQl area?